.github/scripts/lint-pr-title.cjs

```diff
@@ -0,0 +1,104 @@
+/**
+ * This Script can be tested locally with
+ *
+ * export GH_TOKEN=**** GITHUB_REPOSITORY=quisquous/cactbot PR_NUMBER=$NUM
+ * node ./.github/scripts/lint-pr-title.cjs
+ */
+'use strict';
+
+const github = require('@actions/github');
+
+const validScope = [
+  'resources',
+  'ui',
+  'pullcounter',
+  'raidboss',
+  'oopsy',
+  'radar',
+  'eureka',
+  'jobs',
+  'test',
+  'util',
+  'config',
+  'lint',
+  'ci',
+];
+
+const getComment = (title, userName) => `${userName ? '@' + userName : ''} Thanks for your contribution!🚀
+
+Currently your title is: ${title},
+but it should be in the format of \`scope: description\`.
+
+\`scope\` can be any of the following:
+${validScope.map((s) => `  - ${s}`).join('\n')}
+
+------
+This comment was mentained and kept updating by a bot.
```

Comment:
```suggestion
This comment is created and updated by a bot.
```

Some spelling/grammar issues here. I've tried to adjust but I'm not sure exactly what your original intent was.

Reply:
Yeah. I know my English is not very well, basically I write Chinese/Japanese first and then translate it to English (by myself or by machine sometimes).
That's OK, and thank you!